File: 1459a/main.py
def I():
    return input()


def II():
    return int(input())


def MII():
    return map(int, input().split())


def LI():
    return list(input().split())


def LII():
    return list(map(int, input().split()))

a = II()
for _ in range(a):
    bb = II()
    s = I()
    t = I()

    r = b = 0

    for i in range(bb):
        # print(s[i] > t[i], s[i] , t[i])
        if s[i] > t[i]:
            r += 1
        elif s[i] < t[i]:
            b += 1

    if r == b:
        print("EQUAL")
    elif r > b:
        print("RED")
    else:
        print("BLUE")

File: 1459a/main.go
package main

import "fmt"

func solve() {
	var a int
	fmt.Scan(&a)
	var b, c string
	fmt.Scan(&b)
	fmt.Scan(&c)
	r := 0
	k := 0
	for i := 0; i < a; i++ {
		if b[i] > c[i] {
			r++
		} else if b[i] < c[i] {
			k++
		}
	}

	if r == k {
		fmt.Println("EQUAL")
	} else if r > k {
		fmt.Println("RED")
	} else {
		fmt.Println("BLUE")
	}
}
func main() {
	var a int
	fmt.Scan(&a)
	for i := 1; i <= a; i++ {
		solve()
	}
}
